go-runtime/ftl/testdata/go/typeregistry/typeregistry_test.go

```diff
@@ -0,0 +1,68 @@
+package typeregistry
+
+import (
+	"ftl/builtin"
+	"testing"
+
+	"github.com/TBD54566975/ftl/go-runtime/encoding"
+	"github.com/TBD54566975/ftl/go-runtime/ftl"
+	"github.com/TBD54566975/ftl/go-runtime/ftl/ftltest"
+	"github.com/alecthomas/assert"
```

Comment:
assert/v2 .... I do this all the time :(

Reply:
ooo nice catch, thanks!